eisbuk-admin/src/pages/customers/index.tsx

```diff
@@ -54,11 +54,7 @@ const CustomersPage: React.FC = () => {
         {
           // (isLoaded(customers) &&
           !isEmpty(customers) && (
-            <CustomerList
-              {...{ customers }}
-              tableContainerClassName={classes.tableContainer}
-              extended
-            />
+            <CustomerGrid {...{ customers }}></CustomerGrid>
```

Comment:
`<CustomerGrid {props} />` is fine (as opposed to `<CustomerGrid {props}></CustomerGrid>`